content/usgs-packages/Application_Part2.Rmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Application - Part 2, download data"
+date: "9999-05-01"
+author: "Lindsay R. Carr"
+slug: "app-part2"
+image: "img/main/intro-icons-300px/r-logo.png"
+output: USGSmarkdowntemplates::hugoTraining
+parent: Introduction to USGS R Packages
+weight: 2
+draft: true
+---
+
+```{r setup, include=FALSE, warning=FALSE, message=FALSE}
+library(knitr)
+
+knit_hooks$set(plot=function(x, options) {
+  sprintf("<img src='../%s%s-%d.%s'/ title='%s'/>", 
+          options$fig.path, options$label, options$fig.cur, options$fig.ext, options$fig.cap)
+
+})
+
+opts_chunk$set(
+  echo=TRUE,
+  fig.path="static/app-part2/",
+  fig.width = 6,
+  fig.height = 6,
+  fig.cap = "TODO"
+)
+
+knit_hooks$set(addToggle = function(before, options, envir) {
+    if(before) {
+      sprintf('<button class="ToggleButton" onclick="toggle_visibility(\'%1$s\')">Show Answer</button>
+              <div id="%1$s" style="display:none">', opts_current$get('label'))
+    } else {
+      '</div>'
+    }
+})
+
+set.seed(1)
+```
+
+```{r sbtools-auth, echo=FALSE}
+# run vizlab::storeSBcreds() once before this can work
+credList <- readRDS(file.path(path.expand('~'), ".vizlab/sbCreds"))
+un <- rawToChar(credList$username)
+pw <- rawToChar(credList$password)
+sbtools::authenticate_sb(un, pw)
+```
+
+In this section, we are going to use `dataRetrieval` and `geoknife` to get nitrogen, phosphorus, and precipitation data for the sites determined in the [previous section](/usgs-packages/app-part1).
+
+We are walking through the workflow in very distinct chunks, but this will be put together as a single script at the end. If you need a reminder, the code that we used to get the site and 8-digit HUC numbers is available below.
```

Comment:
the only reason I was doing this was so that I could have the right vectors, data frames, etc in place. I didn't know child Rmd documents was a thing? That would be the ideal way to handle it.